docs/freedesktop-quick-reference.rst

```diff
@@ -37,25 +37,54 @@ command, *Type* and *Icon*:
 
     [Desktop Entry]
     Name=Gnome Dictionary
-    Exec=org.gnome.Dictionary
```

Comment:
Did this "`Exec` field is set to the app ID" pattern ever work? From my understanding of this MR, it would've been transformed to 

```
 Exec=/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=org.gnome.Dictionary org.gnome.Dictionary
```

which would fail to run because no command with the name `org.gnome.Dictionary` exists. Other people [have also voiced](https://discourse.flathub.org/t/2322) their non-understanding of this line, with [one person suggesting](https://discourse.flathub.org/t/2322/2) that this would actually work if you did indeed have a `org.gnome.Dictionary` binary/script inside your `org.gnome.Dictionary` Flatpak.

If indeed this is how it worked and why the docs were decided to be written in that way (can someone confirm? @TingPing you commited it?), maybe we should keep the `Exec=org.gnome.Dictionary` example somewhere further below, but with adding the explanation of how it works, instead of replacing it fully with `Exec=gnome-dictionary`. Because `Exec=org.gnome.Dictionary` was in the docs [for the last four years](https://github.com/flatpak/flatpak-docs/blame/master/docs/freedesktop-quick-reference.rst#L40), lots of people have learned it this way, and this pattern is probably found in a lot of Flatpaks in the wild, so it deserves a mention in the docs I think.

Reply:
Sorry for the late reply. This is just what gnome-dictionary's binary is.